enterprise/glassfish.tooling/src/org/netbeans/modules/glassfish/tooling/server/config/JavaSEPlatform.java

```diff
@@ -44,11 +44,23 @@
     v1_5,
     /** JavaSE 1.6. */
     v1_6,
-    /** JavaEE 1.7. */
+    /** JavaSE 1.7. */
     v1_7,
-    /** JavaEE 1.8. */
-    v1_8;
-
+    /** JavaSE 1.8. */
+    v1_8,
+    /** JavaSE 11. */
+    v11,
+    /** JavaSE 12. */
+    v12,
+    /** JavaSE 13. */
+    v13,
+    /** JavaSE 14. */
+    v14,
+    /** JavaSE 15. */
+    v15,
+    /** JavaSE 11. */
```

Comment:
Typo in comment, should be /** JavaSE 16. */

Reply:
OMG, this commit that fixes a typo inserts a new typo!
```suggestion
    /** JavaSE 16. */
```